airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/db/jdbc/JdbcDatabase.kt

```diff
@@ -195,10 +195,10 @@ abstract class JdbcDatabase(protected val sourceOperations: JdbcCompatibleSource
     }
 
     @Throws(SQLException::class)
-    fun queryMetadata(sql: String, vararg params: String): ResultSetMetaData {
+    fun queryMetadata(sql: String, vararg params: String): ResultSetMetaData? {
         unsafeQuery(
                 { c: Connection -> getPreparedStatement(sql, params, c) },
-                { obj: ResultSet -> obj.metaData }
+                { obj: ResultSet -> obj.metaData },
```

Comment:
do we need this extra comma?

Reply:
I think this is something our formatter does